tests/Feature/Modules/Invoices/OrderToUpsellCrudTest.php

```
@@ -0,0 +1,158 @@
+
+        $component = Livewire::actingAs($this->user)
+            ->test(ListOrderToUpsells::class);
+
+        // Should see only 2 records (UPSELLING status)
+        $this->assertEquals(
+            2,
+            Order::where('company_id', $this->company->id)
+                ->where('invoice_status', InvoiceStatus::UPSELLING)
+                ->count()
+        );
+    }
```

PHPMD (2.15.0)

The method it_counts_only_upselling_orders is not named in camelCase. (undefined)

(CamelCaseMethodName)